### DISCUSSION TITLE
Recognise but ignore certain objects

### DISCUSSION THREAD

**Discussion by purpletutle** [blakeblackshear/frigate#3617] [General]
Is there a way to recognize an object but then ignore it and not making a recording? My issue is that I have a lot of cattle in around where my cameras point at the moment. I have the dog object enabled and all the cattle are triggering 100's of clips of them roaming around eating grass. It would be good to detect them as a cow and then just ignore them. 

On another random note, I have the dog objects enabled in the hopes of seeing a fox (I know I have heaps of foxes, wombats and kangeroos going around where the cameras are due to seeing them on the 24/7 recordings) but frigate has never found one, I feel like the night time performance is pretty poor (the cameras can see quite well) is there anything special I need to tweak to improve the night performance?

**Comment by NickM-27** [comment]
1. You could use a max_area filter to block out "dog" that are too big. That or maybe try cat instead of dog
2. Enable `motion -> improve_contrast`

**Reply by purpletutle** [reply]
Interesting, the cattle are also quite often being picked up as humans which is much more of a problem, would playing with the max area on a human help with that even though they are a similar size?

**Reply by NickM-27** [reply]
I don't think so, in frigate 0.11 there's ratio filtering which makes more sense as a human standing is tall while cow is wide
